Violet 12 exercises py.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>I like how you accounted for the possibility of the user passing other variable types! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>